src/command_modules/azure-cli-backup/azure/cli/command_modules/backup/_format.py

```diff
@@ -182,4 +207,5 @@ def transform_wl_policy_list(policy_list):
 
 
 def transform_wl_policy_show(policy_list):
-    return [transform_wl_policy_s(p) for p in policy_list]
\ No newline at end of file
+    if type(policy_list) == list and len(policy_list) != 0 and policy_list[0]['properties']['backupManagementType'] == 'AzureWorkload':
+        return [transform_workload_policy_show(p) for p in policy_list]
```

Comment:
Why are we returning a list for show policy?

Reply:
Change was suggested by you because table formatter was not working otherwise